jekyll/_cci1/test-metadata.md

```diff
@@ -156,6 +157,21 @@ test:
 
 
 
+#### <a name="eslint"></a>ESLint
+
+To output JUnit results from [ESLint](http://eslint.org/), you can use the [JUnit formatter](http://eslint.org/docs/user-guide/formatters/#junit).
+
+A working [circle.yml]( {{ site.baseurl }}/1.0/configuration/) section for testing might look like this:
+
+```
+test:
+  pre:
+    - mkdir -p $CIRCLE_TEST_REPORTS/reports
+    - eslint ./src/ --format junit --output-file $CIRCLE_TEST_REPORTS/reports/eslint.xml
+```
+
+
+
```

Comment:
I just copied the AVA section above with all it's whitespace, it probably doesn't need it